### ISSUE TITLE
PHP 8.1 : Deprecated: Optional parameter $methodArguments declared before required parameter $condition

### ISSUE THREAD

**Issue by jeremyVignelles** [phpfunct/funct#50]
When I run my doctrine migrations, I get this error:

```
Deprecated: Optional parameter $methodArguments declared before required parameter $condition is implicitly treated as a required parameter in /var/www/html/backend/vendor/funct/funct/src/Invoke.php on line 17
```

Faulty code is here:

https://github.com/phpfunct/funct/blob/12f3dcf96f8d3a25765d8e2422feea4294961948/src/Invoke.php#L17

Optional arguments should be placed last.

**Comment by ArtemYurov** [comment]
i have same issue

**Comment by aurimasniekis** [comment]
Should be fixed by #48. Apologies for the long delay!
